### ISSUE TITLE
Ressources pré-fléchées : modifications d'interface

### ISSUE THREAD

**Issue by sabinesafi** [betagouv/recommandations-collaboratives#800]
## Projet sans reco avec ressources pré-fléchées
ex : https://sosponts.recoconseil.fr/project/1506/actions

Supprimer le bouton "Consulter les détails du projet".
Le remplacer par bouton "Examiner les ressources suggérées" et supprimer  la ligne en dessous (le bouton amène à la même page de ressources suggérées)

**Interface attendue :** 

![Image](https://github.com/user-attachments/assets/b0893bc1-a4f0-4486-a3c5-e603a879fdeb)

## Projet avec recos et ressources pré-fléchées
ex : https://sosponts.recoconseil.fr/project/2201/actions

Renommer le 2è bouton : Examiner les ressources suggérées

## Sur la page d'examen des ressources suggérées
ex : https://sosponts.recoconseil.fr/project/2201/suggestions/

- modifier le titre : Recommandations suggérées --> Ressources suggérées automatiquement
- supprimer l'étiquette jaune "brouillon" sur chaque suggestion

## Pour l'instant on fait pas ⬇ 
- Rassembler les suggestions par topic/thématique qui a généré le préfléchage (1)

(1) actuellement on affiche en haut de la page une ligne avec tous les mots-clefs qui ont généré un pré-fléchage (project_topic:xxx). --> Trier les résultats par mot-clef plutôt qu'afficher tout "en vrac". Chaque mot-clef correspondant à l'une des thématiques du projet (thématiques de la page présentation), idéalement afficher ces thématiques, sinon à minima le project_topic:xxx au dessus de chaque groupe de pré-fléchages.

_Affichage actuel des mots-clefs qui ont généré le pré-fléchage :_ 
![Image](https://github.com/user-attachments/assets/9b162362-1d79-4093-a36f-2c47c2399e6e)

_En triant les reco par mots-clefs et en affichant la thématique plutôt que le mot clef :_ 
![Image](https://github.com/user-attachments/assets/32f8b325-e060-43d2-811b-3edb3427e964)

**Comment by Louifine** [comment]
@sabinesafi question, pour ton premier point, je n'ai pas la même interface que toi en prod : 
![Image](https://github.com/user-attachments/assets/07f5e552-9182-422c-abeb-dc5f4f3caa73)
Et dans ce même premier élément, j'ai peut être mal compris mais j'ai l'impression que tu dis que les 2 boutons dont tu parles mènent au même endroit mais ce n'est pas le cas pour moi, le bouton "Consulter les détails du projet" mène vers la page de présentation de l'edl du projet.

**Comment by sabinesafi** [comment]
Hello, est-ce que tu as réussi à clarifier cette histoire d'interface pas la même ?

Je crois qu'on se comprend pas sur l'attendu, je tente de re synthétiser : 
Je veux qu'il y ait 2 boutons (et pas de liens en dessous) - comme sur mon "screenshot" n°1 qui est en fait une maquette : 
- Un bouton Emettre une reco (qui envoie vers le pouse reco)
- Un bouton Examiner les ressources suggérées (qui envoie vers les ressources pré-fléchées)

Dis moi si c'est plus clair @Louifine ?

**Comment by Louifine** [comment]
Yes, du coup plus de bouton qui amène vers les détails du projet (parti edl) ?

**Comment by sabinesafi** [comment]
@Louifine oui c'est ça
